{"name": "Проверка авторизации с некорректыми кредами", "status": "passed", "description": "Попытка авторизации несуществующим пользователем", "start": 1729756427906, "stop": 1729756428776, "uuid": "3e356896-c713-455b-890b-f43cf9ed3803", "historyId": "efd1826884a8b44dcf6655b9b28b6bd4", "testCaseId": "efd1826884a8b44dcf6655b9b28b6bd4", "fullName": "tests.test_auth_user.TestAuthUser#test_auth_with_wrong_accesses", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_user"}, {"name": "subSuite", "value": "TestAuthUser"}, {"name": "host", "value": "MacBook-Pro--Polanski.local"}, {"name": "thread", "value": "39975-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_user"}]}